### ISSUE TITLE
Save model history using callbacks

### ISSUE THREAD

**Issue by ignaciorlando** [ignaciorlando/cnn-dr-kaggle#11]
Based on [this example](https://keras.io/callbacks/#example-recording-loss-history), save model history (using Matplotlib and/or a .CSV file).
